ts/parts/Pointer.ts

```diff
@@ -1112,6 +1112,9 @@ Highcharts.Pointer.prototype = {
             selectionMarker = this.selectionMarker,
             mouseDownX = this.mouseDownX,
             mouseDownY = this.mouseDownY,
+            panningEnabled = isObject(chartOptions.panning) ?
+                chartOptions.panning && chartOptions.panning.enabled :
```

Comment:
Since we already check `isObject`, isn't it safe to simplify to this?
```js
            panningEnabled = isObject(chartOptions.panning) ?
                chartOptions.panning.enabled :
                chartOptions.panning,
```

Reply:
I agree. Will remove it.

Reply:
While tried to simplify it, I've noticed that checking `chart.panning` is done because of TypeScript rules. This property can also be type of `undefined`, so after doing it as you suggested, TS thrown the error, because we're trying to refer something what can be undefined (by `chartOptions.panning.enabled`). In short, TS does not know what `isObject` is actually doing, and shows the error in this case. Maybe @bre1470 would have any idea how to achieve it in other way?

Reply:
Okay, we can leave it as is for now.